src/main/kotlin/no/skatteetaten/aurora/boober/service/DeployService.kt

```diff
@@ -248,5 +252,17 @@ class DeployService(
 
         return changed
     }
+
+    private fun createImageStream(openShiftResponses: List<OpenShiftResponse>): ImageStream? {
+        val imageStream: OpenShiftResponse = openShiftResponses.find { it.responseBody?.openshiftKind == "imagestream" }
+                ?: return null
```

Comment:
her synes jeg en 
`openShiftResponses.find { it.responseBody?.openshiftKind == "imagestream" }
  ?.let{ jacksonObjectMapper...}`

er på sin plass